subworkflows/local/eggnog.nf

```diff
@@ -10,17 +10,14 @@ workflow EGGNOG {
     take:
     faa
     collect_fcs
-    dbpath
 
     main:
     ch_versions = Channel.empty()
 
-    ch_dbpath = Channel.fromPath(dbpath, checkIfExists: true)
-
     EGGNOG_DOWNLOAD()
     ch_versions = ch_versions.mix ( EGGNOG_DOWNLOAD.out.versions )
 
-    EGGNOG_MAPPER ( faa, ch_dbpath, EGGNOG_DOWNLOAD.out.eggnog_db )
+    EGGNOG_MAPPER ( faa, EGGNOG_DOWNLOAD.out.all )
```

Comment:
Does this work? I thought you'd have to specify the individual files, as in Mahesh's example, rather than this.
There is an advantage in specifying all individual files also by being more explicit about which files are actually needed.

Reply:
I tried this way and it works. If you think it is best to specify each file I can do it. 
Should I change `EGGNOG_DOWNLOAD` then? as he showed in the example...

Reply:
`EGGNOG_DOWNLOAD` is already emitting the required files, one by one. I'd feel safer with having explicit files as input to the mapping process.

Reply:
@erikrikarddaniel